ui/ui-frontend/package.json

```diff
@@ -173,14 +158,11 @@
     "karma-junit-reporter": "2.0.1",
     "karma-spec-reporter": "0.0.36",
     "lint-staged": "13.2.3",
-    "moment-locales-webpack-plugin": "^1.2.0",
     "ng-packagr": "^17.3.0",
     "prettier": "3.3.2",
-    "protractor": "^7.0.0",
```

Comment:
Il faudrait supprimer la section "Running end-to-end tests" du README.md

Reply:
C'est fait.